charts/apisix/values.yaml

```diff
@@ -86,6 +86,12 @@ apisix:
     # -- Enable the proxy protocol to the upstream server
     upstream: true
 
+  # -- Enable Data Encryption
+  data_encryption:
+    enabled: false
+    # -- An array of 16 character strings used to encrypt/decrypt fields with AES-128-CBC
```

Comment:
I reworded the description from https://github.com/apache/apisix/pull/8403/files#diff-8d872babc717e9d733641b56bfc530ef98751fbe4e68f08d79b2b83109c22fffR130